swagger.json

```diff
@@ -1121,6 +1121,77 @@
             }
           }
         },
+        "/fleet/drivers/{driver_id}/hos_daily_log": {
+            "patch": {
+              "consumes": [
+                "application/json"
+              ],
+              "description": "Modify details, such as trailer number or shipping ID, for a driver's daily log",
+              "operationId": "edit_fleet_drivers_hos_daily_log_details",
+              "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "description": "ID (integer) of the driver with HOS logs.",
+                  "in": "path",
+                  "name": "driver_id",
+                  "required": true,
+                  "type": "string"
+                },
+                {
+                  "in": "body",
+                  "name": "editHosDailyLogParams",
+                  "required": true,
+                  "schema": {
+                    "properties": {
+                      "timestamp": {
+                        "description": "A specified timestamp within the bounds of a driver's daily log, specified in milliseconds UNIX time.",
+                        "example": 1462881998034,
+                        "type": "integer"
+                      },
+                      "shippingId": {
+                        "description": "Shipping Id to add to driver's daily log.",
+                        "example": 101,
+                        "format": "int64",
+                        "type": "integer"
+                      },
+                      "trailerName": {
+                        "description": "Trailer name to add to driver's daily log",
```

Comment:
Should add restriction that it can only be a max of 255 characters

Reply:
there is also a trick here that, If they are appending 255 chars to a field that already has 100 chars the last 100 chars from the request will be ignored.

We should talk with PM about this if we want to throw and error for scenarios like this or just truncate on our side which means the user is not aware that this happened on the backend